.gitignore

```diff
@@ -11,3 +11,5 @@ build
 coverage
 node_modules
 test/benchmarks/.tmp
+coverage.lcov
```

Comment:
This isn't really related to this PR right?

Reply:
That's a good point, I added it since I did run the same scripts locally to reproduce the same steps as the CI and that particular file was created on the fly.  No strong opinion to revert it if required.